<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>309</width>
    <height>228</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:white;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="barParentLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QPushButton" name="hitButton">
          <property name="styleSheet">
           <string notr="true">background-color:rgb(180,0,0);
color:white;
border:none;
outline:none;
padding:10px;
font-weight:700;
border-radius: 16;</string>
          </property>
          <property name="text">
           <string>Hit</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="healButton">
          <property name="styleSheet">
           <string notr="true">background-color:rgb(0,180,0);
color:white;
border:none;
outline:none;
padding:10px;
font-weight:700;
border-radius:16;</string>
          </property>
          <property name="text">
           <string>Heal</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="limitLabel">
        <property name="styleSheet">
         <string notr="true">font-size:20px;
font-weight:700;
color:white;
background-color: #444;
border-radius:8;
padding-left:10px;
margin-top:20px;</string>
        </property>
        <property name="text">
         <string>50</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="limitSlider">
        <property name="value">
         <number>50</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>309</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
